docs/ipdk-tofino.md

```diff
@@ -0,0 +1,115 @@
+IPDK on tofino
+===============
+
+# Overview
+This document describes how to build and run IPDK on Tofino.
+
+# Requirements
+- Docker (Ubuntu 20.04 base image)
+- Intel P4Studio
+- Git
+
+# Setup
+The build and execution of infrap4d uses docker.
+
+## Docker
+
+### Pull the Ubuntu 20.04 docker base image
+> docker pull ubuntu:20.04
+
+### Create container
+> docker run --privileged --cap-add=ALL -it --name infrap4d --network host -d ubuntu:20.04 /bin/bash
+
+## Git
+
+### Clone IPDK networking-recipe
+> git clone --recursive git@github.com:ipdk-io/networking-recipe.git infrap4d
```

Comment:
Please use a different name for the superproject than `infrap4d`.

`infrap4d` is just one of the artifacts the recipe generates. It also generates `ovs` and the client utilities, and it will eventually include kubernetes.

I've been using `ipdk.recipe` as my local repository name.

Reply:
Done